sdk/src/Core/Amazon.Runtime/AmazonServiceClient.cs

```diff
@@ -45,6 +45,12 @@ public abstract class AmazonServiceClient : IDisposable
         protected RuntimePipeline RuntimePipeline { get; set; }
         public IClientConfig Config => _config;
         private readonly ClientConfig _config;
+
+        /// <summary>
+        /// Credentials explicitly specified when constructing the client.
+        /// </summary>
+        protected internal AWSCredentials DefaultAWSCredentials { get; private set; }
```

Comment:
To avoid having 2 properties called `DefaultAWSCredentials` that mean slightly different things what do you think of `ExplicitAWSCredentials`? The meaning that this is set when a user explicitly passes in credentials to the service client.

Reply:
Updated in https://github.com/aws/aws-sdk-net/pull/3958/commits/d3c3771378f6dca00ecaee25c18708817c1e07a1